SocketRocket/SRWebSocket.m

```diff
@@ -534,6 +534,14 @@ - (void)_initializeStreams;
     
     CFStreamCreatePairWithSocketToHost(NULL, (__bridge CFStringRef)host, port, &readStream, &writeStream);
     
+    //If voip is set in UIBackgroundMode, set apropriate properties on read and write stream
+    NSArray* backgroundModes = [[[NSBundle mainBundle] infoDictionary] objectForKey:@"UIBackgroundModes"];
```

Comment:
Agreed with @dfed on this. We should probably add a category to NSURLRequest using the whole `+[NSURLProtocol propertyForKey:inRequest:]` set of functions to control this.